### ISSUE TITLE
File Transfer Recording not able to playback

### ISSUE THREAD

**Issue by ONBET** [Ylianst/MeshCentral#3424]
Meshcentral: Version 0.9.62

Terminal session and Desktop Session is working fine but the File Transfer recording not able to playback. Even I try open player manually select the file transfer recording file to playback but no luck still can't. 

Is it bug ?

**Comment by Ylianst** [comment]
The file session can be recorded, but I never wrote a player that could view this session type. I am not sure what use it would have, I guess it would show what file operations have been made.

**Comment by ONBET** [comment]
Do you mean that the file session can be recorded but the player for file session video is not yet ready yet?  Is there any video player can play it ? Otherwise hope to see it soon in new release

**Comment by ONBET** [comment]
Yes to show what file operations have been made for audit purpose and for security to ensure our IT staff no abuse the power of remote

**Comment by ONBET** [comment]
@Ylianst 

Is there any update on this? I just tested 0.9.96 still not able to playback the file transfer recording.
